ain/utils.py

```diff
@@ -0,0 +1,114 @@
+import re
+import json
+import time
+from typing import Any, Union
+from Crypto.Hash import keccak as _keccak
+from coincurve import PrivateKey
+from ain.types import ECDSASignature, TransactionBody
+
+
+def getTimestamp() -> int:
+    """
+    Gets a current timestamp.
+    """
+    return int(time.time() * 1000)
+
+
+def isValidAddress(address: str) -> bool:
+    pattern = re.compile("0x[0-9a-fA-F]{40}")
+    return pattern.fullmatch(address) is not None
+
+
+def isHexString(value: str) -> bool:
+    pattern = re.compile("0x[0-9a-fA-F]*")
+    return pattern.fullmatch(value) is not None
+
+
+def isHexPrefixed(value: str) -> bool:
+    return value[0:2] == "0x"
+
+
+def stripHexPrefix(value: str) -> str:
+    if isHexPrefixed(value):
+        value = value[2:]
+    return value
+
+
+def padToEven(value: str) -> str:
+    if len(value) % 2 == 1:
+        value = "0" + value
+    return value
+
+
+def toBuffer(input: Any) -> bytes:
```

Comment:
This method seems to be a little different from the corresponding method in ain-js. Is it not fully implemented yet or did some of the cases get omitted because they are unnecessary / already covered?

Reply:
It isn't fully implemented yet, I'll add TODO.

Reply:
Done